tokens/global/typography/lineHeights.js

```diff
@@ -0,0 +1,56 @@
+import typography from './core.js';
+import sizes from './fontSizes.js';
+
+const base = typography.base;
+const fontSizes = sizes.fontSizes;
+const headingSizes = ['h1', 'h2', 'h3', 'h4', 'h5', 'h6'];
+const octave = 6;
+
+/**
+ * Calculates line height as a percentage based on font size.
+ *
+ * This function:
+ * - Takes font size as input and scales it relative to base font size.
+ * - Determines number of baseline octaves required to fit font size.
+ * - Adds a small additional octave to ensure adequate spacing.
+*/
+const calculateLineHeight = (fontSize) => {
+  const fontSizeValue = parseFloat(fontSize) / base.fontSize.value;
```

Comment:
This function, if the `fontSize` is set to `base.fontSize.value` returns the same value as
```
base.lineHeight.value * 100
```

but we can simply skip all the other computation and just return the value if we're computing the line height for the base font size like this:

```suggestion
  const calculateLineHeight = (fontSize) => {
  if( fontSize === base.fontSize.value ) {
    return base.lineHeight.value * 100
  }
  const fontSizeValue = parseFloat(fontSize) / base.fontSize.value;
```

Reply:
I will rewrite the line height calculation to implement the new typography. Would you be okay with leaving it as is for now?

Reply:
Yes sounds good to me!